SingularityUI/app/components/common/modalButtons/DeletePendingTaskModal.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+
+import { DeletePendingOnDemandTask } from '../../../actions/api/tasks';
+
+import FormModal from '../../common/modal/FormModal';
+
+class DeletePendingTaskModal extends Component {
+  static propTypes = {
+    taskId: PropTypes.string.isRequired,
+    requestType: PropTypes.string.isRequired,
+    deletePendingTask: PropTypes.func.isRequired,
+    then: PropTypes.func
+  };
+
+  show() {
+    this.refs.deletePendingTaskModal.show();
+  }
+
+  render() {
+    return (
+      <FormModal
```

Comment:
Do we not already have a confirm modal? Feels bad doing it as a `FormModal` with no fields.

Reply:
Ah yes, we do have one! I'll swap it out.